org.virtualxt.VirtualXT.yml

```diff
@@ -0,0 +1,88 @@
+app-id: org.virtualxt.VirtualXT
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: virtualxt.freedos
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=home
+  - --filesystem=/dev
+  - --filesystem=/media
+  - --filesystem=/run/media
+  - --device=all
+build-options:
+  env:
+    - GOROOT=/usr/lib/sdk/golang
+
+modules:
+  - name: SDL2
+    buildsystem: autotools
+    config-opts:
+      - --disable-video-wayland
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share
+    sources:
+      - type: archive
+        url: https://www.libsdl.org/release/SDL2-2.0.20.tar.gz
+        sha256: c56aba1d7b5b0e7e999e4a7698c70b63a3394ff9704b5f6e1c57e0c16f04dd06
+
+  - name: virtualxt
+    buildsystem: simple
+    build-commands:
+    
+      # Fix for 0.6.1. Remove later.
+      - cp _go.mod go.mod
+      - cp _go.sum go.sum
+      - cp _modules.txt vendor/modules.txt
+
+      - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/lib
+      - export C_INCLUDE_PATH=$C_INCLUDE_PATH:/include
```

Comment:
I'm not completely sure here what the env do, but there is no `/lib` not `/include`

Reply:
Hmm.. yea.. that looks like it does nothing. Removing